src/utils.js

```diff
@@ -3,9 +3,9 @@
 import * as React from 'react';
 
 export const createRange = (start: number, end: number): Array<number> =>
-  [...Array(1 + end - start).keys()].map(
```

Comment:
thats weird as hell, but for some reason this implementation didn't work for a few cases when react-native debug mode is disabled O_o

Reply:
The new solution is better anyways.